File: frontend/web_ccadmin_lt/src/app/enterprise/product/model/dto/ProductConfigStoreUpdateDto.ts
import { ProductConfigEntity } from "../entity/ProductConfigEntity";

export class ProductConfigStoreUpdateDto {
    public ProductCod: string = "";
    public StoreCod: string = "";
    public StoreCodList: string[] = [];
    public ApplyAllStores: boolean = false;
    public config: ProductConfigEntity = new ProductConfigEntity();
}

File: backend/ws_wa_store_ccadmin/src/main/java/com/ccadmin/app/product/model/dto/ProductConfigStoreUpdateDto.java
package com.ccadmin.app.product.model.dto;

import com.ccadmin.app.product.model.entity.ProductConfigEntity;
import com.ccadmin.app.product.model.entity.ProductTaxConfigEntity;

import java.util.ArrayList;
import java.util.List;

public class ProductConfigStoreUpdateDto {
    public String ProductCod;
    public String StoreCod;
    public List<String> StoreCodList = new ArrayList<>();
    public boolean ApplyAllStores;
    public ProductConfigEntity config = new ProductConfigEntity();
    public List<ProductTaxConfigEntity> TaxConfigList = new ArrayList<>();
}
